tests/sentry/lang/native/test_appconnect.py

```diff
@@ -167,7 +168,7 @@ class TestDownloadDsyms:
     def client(self) -> appconnect.AppConnectClient:
         return appconnect.AppConnectClient(
             app_id="honk",
-            api_credentials=appconnect.appstore_connect.AppConnectCredentials(
+            api_credentials=appstore_connect.AppConnectCredentials(
```

Comment:
this was depending on import side-effects, new mypy correctly identified this wasn't being re-exported